### PR TITLE
Fix update password

### PR DESCRIPTION
**Pull request by bsoniam**

### CODE REVIEW

File: pkg/account/component.go [hunk 1]
```diff
@@ -75,6 +75,10 @@ func (c *component) UpdatePassword(ctx context.Context, currentPassword, newPass
 	}
 
 	_, err := c.keycloakAccountClient.UpdatePassword(accessToken, realm, currentPassword, newPassword, confirmPassword)
+	if err != nil {
+		c.logger.Warn("err", err.Error())
+		return err
```
Comment (fperot74) [inline pkg/account/component.go]: You should return nil in line 97

Comment (bsoniam) [reply pkg/account/component.go]: But err should be nil in that case, no?

Comment (fperot74) [reply pkg/account/component.go]: Yes but writing `return err` let think that the returned value could be different

Comment (bsoniam) [reply pkg/account/component.go]: True, you're right. I will change this

**Pull request merged**